web-components/src/components/form/form.stories.tsx

```diff
@@ -108,3 +112,13 @@ export const basketTakeForm = (): JSX.Element => (
     onSubmit={async () => alert('submit')}
   />
 );
+
+export const recipientsForm = (): JSX.Element => (
```

Comment:
to be more consistent with the story name in #940:
```suggestion
export const creditBatchRecipientsForm = (): JSX.Element => (
```

Reply:
@victordelval up